tests/utils/scrollMonitor.test.ts

```diff
@@ -0,0 +1,44 @@
+import ScrollMonitor from '../../src/utils/scrollMonitor';
+
+describe('#ScrollMonitor', async () => {
+  describe('#subscribe', () => {
```

Comment:
If you are doing the betterspecs names, Static methods are `.` and instance methods are `#`
See --> http://www.betterspecs.org/
```suggestion
  describe('.subscribe', () => {
```

Reply:
You are correct sir. Addressing now.